tests/results/result_actions/OpenEditorActionTest.py

```diff
@@ -191,3 +169,68 @@ def test_environ_editor(self):
         action.apply(result, file_dict, diff_dict)
 
         os.environ = old_environ
+
+    def test_open_files_at_position_unknown_editor(self):
+        uut = OpenEditorAction()
+        result_mock = Result.from_values(
+            'test', '', self.fa, line=12, column=8,
+        )
+        with unittest.mock.patch('subprocess.call') as call:
+            uut.apply(result_mock, {self.fa: ''}, {}, editor='unknown_editor')
+            call.assert_called_with(
+                ['unknown_editor', self.fa]
+            )
+
+    def test_open_files_at_position_subl(self):
+        uut = OpenEditorAction()
+        result_mock = Result.from_values(
+            'test', '', self.fa, line=12, column=8,
+        )
+        with unittest.mock.patch('subprocess.call') as call:
+            uut.apply(result_mock, {self.fa: ''}, {}, editor='subl')
+            call.assert_called_with(
+                ['subl', '--wait', '{0}:12:8'.format(self.fa)],
+                stdout=subprocess.PIPE
+            )
+
+    def test_open_files_at_position_vim(self):
+        uut = OpenEditorAction()
+        result_mock = Result.from_values(
+            'test', '', self.fa, line=12, column=8,
+        )
+        with unittest.mock.patch('subprocess.call') as call:
+            uut.apply(result_mock, {self.fa: ''}, {}, editor='vim')
+            call.assert_called_with(
+                ['vim', self.fa, '+12']
+            )
+
+    def test_open_files_at_position_no_position(self):
+        uut = OpenEditorAction()
+        result_mock = Result.from_values(
+            'test', '', self.fa, line=None, column=None,
+        )
+        with unittest.mock.patch('subprocess.call') as call:
+            uut.apply(result_mock, {self.fa: ''}, {}, editor='subl')
+            call.assert_called_with(
+                ['subl', '--wait', '{0}:1:1'.format(self.fa)],
+                stdout=subprocess.PIPE
+            )
+
+
+def test_build_editor_call_args_spaced_filename():
+    uut = OpenEditorAction()
+    editor_info = {
+        'args': '--bar --baz',
+        'file_arg_template': '{filename} +{line}'
+    }
+    filenames = {
+        'foo and bar.py': {
+            'filename': 'foo and bar.py',
+            'line': 10,
+            'column': 6
+        }
+    }
+    assert (
+        uut.build_editor_call_args('foo', editor_info, filenames) ==
+        ['foo', '--bar', '--baz', 'foo and bar.py', '+10']
+    )
```

Comment:
No. I tried some with differing results.

Most with the `<filename>:<line>:<column>` syntax like sublime can, gedit and xed with the `<filename> +<line>` syntax will only use one of the provided lines for all.

I see three options:

* Leave as is. This will result in files opened at the wrong position in some circumstances
* Disable the "open at position" feature if multiple files are to be opened
* Find out for each editor how it behaves and add a `can_open_multiple_files_at_position` key to the `editor_infos`. 

What do you think?